### ISSUE TITLE
Probes available API call is empty

### ISSUE THREAD

**Issue by justb4** [geopython/GeoHealthCheck#167]
For example: http://demo.geohealthcheck.org/api/v1.0/probes-avail/ (get all `Probes` available in GHC instance) gives empty JSON response. Also for 
http://demo.geohealthcheck.org/api/v1.0/probes-avail/OGC:WMS (all `Probes` for WMS)

Only gives result with specific `Resource` id, like
http://demo.geohealthcheck.org/api/v1.0/probes-avail/OGC:WMS/14

Expected is a JSON struct with available `Probes` with parameters for instance like: 

```
{
  "GeoHealthCheck.plugins.probe.http.HttpGet": {
    "AUTHOR": "GHC Team", 
    "CHECKS_AVAIL": {
      "GeoHealthCheck.plugins.check.checks.ContainsStrings": {
        "AUTHOR": "GHC Team", 
        "DESCRIPTION": "HTTP response contains all (comma-separated) strings specified", 
        "NAME": "Response contains strings", 
        "PARAM_DEFS": {
          "strings": {
            "default": null, 
            "description": "The string text(s) that should be contained                  in response (comma-separated)", 
            "range": null, 
            "required": true, 
            "type": "stringlist"
          }
        }
      }, 
      "GeoHealthCheck.plugins.check.checks.HttpHasContentType": {
        "AUTHOR": "GHC Team", 
        "DESCRIPTION": "HTTP response has specific Content-Type", 
        "NAME": "Has specific Content-Type", 
        "PARAM_DEFS": {
          "header_name": {
            "default": null, 
            "description": "The HTTP header name", 
            "range": null, 
            "required": true, 
            "type": "string", 
            "value": "content-type"
          }, 
          "header_value": {
            "default": null, 
            "description": "The HTTP header value", 
            "range": null, 
            "required": true, 
            "type": "string"
          }
        }
      }, 
      "GeoHealthCheck.plugins.check.checks.HttpStatusNoError": {
        "AUTHOR": "GHC Team", 
        "DESCRIPTION": "Response should not contain a HTTP 400 or 500 range Error", 
        "NAME": "HTTP status should not be errored", 
        "PARAM_DEFS": {}
      }, 
      "GeoHealthCheck.plugins.check.checks.NotContainsStrings": {
        "AUTHOR": "GHC Team", 
        "DESCRIPTION": "\n        HTTP response does not contain any of the\n        (comma-separated) strings specified\n        ", 
        "NAME": "Response NOT contains strings", 
        "PARAM_DEFS": {
          "strings": {
            "default": null, 
            "description": "The string text(s) that should NOT be\n                contained in response (comma-separated)", 
            "range": null, 
            "required": true, 
            "type": "stringlist"
          }
        }
      }
    }, 
    "DESCRIPTION": "Simple HTTP GET on Resource URL", 
    "NAME": "HTTP GET Resource URL", 
    "PARAM_DEFS": {}, 
    "REQUEST_HEADERS": {}, 
    "REQUEST_METHOD": "GET", 
    "REQUEST_TEMPLATE": "", 
    "RESOURCE_TYPE": "*:*"
  }, 
  "GeoHealthCheck.plugins.probe.http.HttpGetQuery": {
    "AUTHOR": "GHC Team", 
    "CHECKS_AVAIL": {
.
.

```

**Comment by justb4** [comment]
Ok, works (again): see http://dev.geohealthcheck.org/api/v1.0/probes-avail/ and
http://dev.geohealthcheck.org/api/v1.0/probes-avail and http://dev.geohealthcheck.org/api/v1.0/probes-avail/OGC:WMS.

**Comment by justb4** [comment]
Fixed and merged into master via #169.
